src/exts/tvdb_info/main.py

```diff
@@ -118,6 +119,7 @@ def __init__(self, bot: Bot) -> None:
         required=False,
     )
     @option("by_id", input_type=bool, description="Search by tvdb ID.", required=False)
+    @rate_limited(key=lambda self, ctx: f"{ctx.user}", limit=2, period=8, update_when_exceeded=True, prefix_key=True)
```

Comment:
I had many issues before with applying custom decorators to slash commands. I assume you tested this, with the options (by_id and type) as well